.github/workflows/scale-out.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 
 jobs:
   IoTDB-Tag-Test:
```

Comment:
Tag-Test seems to have similar testing objectives as session test ad rest test in the same file. So why not structure them similarly?